demomvp/data/source/DbStudentManager.java

```diff
@@ -0,0 +1,102 @@
+package com.duan1.nhom4.demomvp.data.source;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.duan1.nhom4.demomvp.data.Callback;
+import com.duan1.nhom4.demomvp.data.model.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DbStudentManager extends SQLiteOpenHelper {
+    private static final String TABLE_NAME = "STUDENT";
+    private static final String COLUMN_NO = "NUMBER";
+    private static final String COLUMN_NAME = "NAME";
+    private static final String COLUMN_BIRTH = "BIRTHDAY";
+    private static final String COLUMN_CLASS = "CLASSSTUDENT";
+
+    private static final String CREATE_TABLE =
+            "CREATE TABLE " + TABLE_NAME + " ("
+                    + COLUMN_NO + " INTEGER PRIMARY KEY AUTOINCREMENT, "
+                    + COLUMN_NAME + " TEXT, "
+                    + COLUMN_BIRTH + " DATETIME, "
+                    + COLUMN_CLASS + " TEXT )";
+
+    private static final int DATABASE_VERSION = 1;
+    private static final String DATABASE_NAME = "Student_db";
+
+    private static DbStudentManager sInstance;
+
+    public static DbStudentManager getInstance(Context context) {
+        if (sInstance == null) {
+            sInstance = new DbStudentManager(context);
+        }
+        return sInstance;
+    }
+
+    private DbStudentManager(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        db.execSQL(CREATE_TABLE);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        onCreate(db);
+    }
+
+    public void insertStudent(String name, String birthDay, String classStudent) {
+        SQLiteDatabase db = getWritableDatabase();
+        db.beginTransaction();
+        try {
+            ContentValues values = new ContentValues();
+            values.put(COLUMN_NAME, name);
+            values.put(COLUMN_BIRTH, String.valueOf(birthDay));
+            values.put(COLUMN_CLASS, classStudent);
+            db.insertOrThrow(TABLE_NAME, null, values);
+            db.setTransactionSuccessful();
+        } catch (Exception e) {
+            throw e;
+        } finally {
+            db.endTransaction();
+        }
+    }
+
+    public void getAllStudent(Callback<List<Student>> callback) {

```

Comment:
```suggestion
    public void getStudents(Callback<List<Student>> callback) {
```
e nên tạo các method với số nhiều ntn